app.rb

```diff
@@ -0,0 +1,52 @@
+require_relative "capitals"
+states = data[:states]
+
+puts "Yo, Yo, Yo! Welcome to the State Capitals Quiz!"
+puts "To play: type GO"
+
+answer = gets.chomp
+
+
+
+states = states.shuffle!
+ def get_user_input prompt
+   puts prompt
+   gets.chomp
+ end
+
+
+if answer == "GO"
+  states.map do |state|
```

Comment:
this map method works identical to the each method in this case since you aren't storing the return value